packages/clerk-js/src/ui/utils/test/mockHelpers.ts

```diff
@@ -30,6 +30,46 @@ export const mockClerkMethods = (clerk: LoadedClerk): DeepJestMocked<LoadedClerk
   mockMethodsOf(clerk.client.signUp);
   clerk.client.sessions.forEach(session => {
     mockMethodsOf(session);
+    session.isAuthorized = jest.fn(args => {
```

Comment:
❓ Why do we need to mock this helper? Should we use the actual method and test it?

Reply:
The line above assigns jest.fn() to all methods of the object, i basically restore it. 

Tests already exists in Session.test.tsx, but re-implementing the function here was necessary for me to test the usage inside the components, as they are depending on it now.

Reply:
Do u think we should change `mockMethodsOf` to accept a second argument that will skip assigning jest.fn to specified properties ?